src/Stratis.Bitcoin/P2P/Peer/NetworkPeer.cs

```diff
@@ -637,13 +661,17 @@ private void FireFilters(IEnumerator<INetworkPeerFilter> enumerator, Payload pay
                 }
                 catch (Exception ex)
                 {
-                    this.TraceCorrelation.LogInside(() => NodeServerTrace.Error("Unhandled exception raised by a node filter (OnSendingMessage)", ex.InnerException), false);
+                    this.logger.LogError("Exception occurred: {0}", ex.InnerException.ToString());
```

Comment:
The following error logs only have `Exception occurred` when previously there was a more detailed message is that intentional?

Reply:
yes, because due to how our NLog is set up, that information is included implicitly